mavros_extras/src/plugins/distance_sensor.cpp

```diff
@@ -302,7 +309,7 @@ DistanceSensorItem::Ptr DistanceSensorItem::create_item(DistanceSensorPlugin *ow
 		}
 
 		// orientation check
-		pnh.param("orientation", p->orientation, -1);
+		pnh.param("orientation", orient_value, -1);
```

Comment:
`p->orientation` has to be a string, but how do I treat it as an integer? Isn't `p->orientation` same as https://github.com/mavlink/mavros/pull/342/files#diff-2adf7423447a0e5f4880b56161efdda9R51?